stock_deposit/README.rst

```diff
@@ -0,0 +1,86 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+=============
+Stock deposit
+=============
+
+This module extends standard WMS to manage customers deposit locations by
+warehouse
```

Comment:
You should mention that deposited stocks can't be selected in regular delivery orders unless they have been returned.

Also tell about the inventory valuation.